<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanningWidget</class>
 <widget class="QWidget" name="ScanningWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanningWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mapScaleLabel">
       <property name="text">
        <string>Tỷ lệ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mapScaleComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="scanningDirLabel">
       <property name="text">
        <string>Thư mục:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scanDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseScanDirButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Chọn...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Quét</string>
       </property>
       <property name="icon">
        <iconset resource="MainWindow.qrc">
         <normaloff>:/icons/scan.svg</normaloff>:/icons/scan.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="folderProgressLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="storeDirLabel">
       <property name="text">
        <string>Nơi lưu trữ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="storeDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseStoreDirButton">
       <property name="text">
        <string>Chọn...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progressPathLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveTableButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Lưu bảng</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="storeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Nhập kho</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
